source/datasets/parking.rst

```diff
@@ -1,36 +1,11 @@
 .. parking
 
-..
-   This dataset contains two types of data:
-
-   * Parking stations - off street parking data (currently for the cities
-     of Bolzano, Merano, Trento, Rovereto), with slot availability and
-     predictions
-
-   * Parking sensors - on street parking data (currently Bolzano, soon
-     Merano). Single parking slots on streets, can be within a virtual
-     area
-
-   .. csv-table::
-
-      "Output", "JSON, mime-type application/json"
-      "E-mail contact", "|contact|"
-      "API version", ":strike:`v1` |deprecated|, v2"
-      ":literal:`Station Type`", "`ParkingStation
-      <https://mobility.api.opendatahub.bz.it/v2/flat/ParkingStation>`_,
-      `ParkingSensor
-      <https://mobility.api.opendatahub.bz.it/v2/flat/ParkingSensor>`_"			   
-      "Use cases and info", "https://parking.bz.it/,
-      https://mobility.meran.eu/, https://analytics.opendatahub.bz.it/,
-      https://mobility.bz.it/"			   
-      "Sources", "Municipalities of Bolzano, Merano, Trento, and
-      Rovereto"
-
```

Comment:
@stefanodavid @rcavaliere Why do we remove these information?

Reply:
@Piiit I removed a commented region, that I left behind during a previous editing, so no information is lost.